packages/core/core/src/types.ts

```diff
@@ -28,6 +28,19 @@ export interface UseProduct<PRODUCTS, PRODUCT_SEARCH_PARAMS> {
   [x: string]: any;
 }
 
+export interface UseForgotPasswordErrors {
+  result: Error;
```

Comment:
What about `change: Error;`?

Reply:
As these two methods cannot be used at the same moment (reset in Login Page, change in reset Password page) I decided that there is no need to craete separate errors just for that. Do you think it is necessary to have both methods have separate errors?